packages/analytics/dashboard-renderer/src/components/DashboardRenderer.cy.ts

```diff
@@ -508,21 +498,21 @@ describe('<DashboardRenderer />', () => {
       props,
       global: {
         provide: {
-          [INJECT_QUERY_PROVIDER]: mockQueryProvider({ skuFeatureFlag: true }),
+          [INJECT_QUERY_PROVIDER]: mockQueryProvider({ }),
         },
       },
     }).then(() => {
       // Extra calls may mean we mistakenly issued queries before knowing the timeSpec.
-      cy.get('@fetcher').should('have.callCount', 5)
+      cy.get('@fetcher').should('have.callCount', 2)
```

Comment:
@adorack if I run this test with `.only`, it indeed intercepts this call five times.

But if i run in sequence, the `fetcher` only gets called twice 🤷

Reply:
I can look at this after the other issues are resolved.